packages/paste-core/components/typography/src/Paragraph.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+import {ParagraphProps} from './types';
+
+const handlePropValidation = ({children}: ParagraphProps): void => {
+  if (children == null) {
+    throw new Error(`[Paste: Typography Paragraph] Must have non-null children.`);
+  }
+};
+
+const Paragraph: React.FC<ParagraphProps> = props => {
```

Comment:
I'd like us to also start giving our components a displayName manually.  This makes it so you can retrieve the component type while iterating through react children.  

So right before the export just add
`Paragraph.displayName = 'Paragraph';`

React.FC types it correctly so it should just work.

Reply:
Had to look up what adding a `displayName` does, but now I know. Added it.